### DISCUSSION TITLE
0.6.5 released

### DISCUSSION THREAD

**Discussion by abartesaghi** [nextpyp/.github#106] [Announcements]
This release contains some bug fixes and minor improvements. See the [changelog](https://nextpyp.app/files/pyp/0.6.5/docs/changelog.html) for details.

To upgrade, follow the instructions described in the [docs](https://nextpyp.app/files/pyp/0.6.5/docs/install/install-web.html#upgrading-to-a-new-version).

**Comment by asarnow** [comment]
Hi Alberto, I upgraded and restarted but the version displayed is still 0.6.4. Is that just a display string or is there a real issue with the file version?

**Reply by abartesaghi** [reply]
You are right, the version number was not displayed correctly in the UI. We've patched the necessary files to fix this problem. As far as we can tell, this should not have affected the functionality of nextPYP in any way. After re-installing 0.6.5 it should show the correct version number. Thanks for reporting this.
